OpenWrt Rules.md

```diff
@@ -34,14 +34,14 @@ _**Voting**_
   The proposal must clearly describe the suggested changes
   and include a specific deadline for when the voting period will end.
   A simple approval is required.
+* All active members who voted in the past 6 months before the new vote was started are considered active voters. If less than 3 votes happened in the past 6 months the last 3 votes are considered to determine the active voters.
```

Comment:
While all lines in the document before this addition are at most 80 characters long, this line is much longer and doesn't fit the existing formatting style. I suggest to keep the limit of 80 characters per line also here.

Reply:
fixes it